src/app/pages/HomePage/ParaTimeSelector/Graph/index.tsx

```diff
@@ -68,13 +68,14 @@ const GraphCmp: FC<GraphProps> = ({
   setSelectedGraphEndpoint,
 }) => {
   const navigate = useNavigate()
-
-  const onSelectGraphEndpoint = (graphEndpoint: GraphEndpoint) => {
+  const onSelectEmeraldBlocks = () => onSelectGraphEndpoint(GraphEndpoint.EMERALD, 'blocks')
+  const onSelectEmeraldTxs = () => onSelectGraphEndpoint(GraphEndpoint.EMERALD, 'transactions')
+  const onSelectGraphEndpoint = (graphEndpoint: GraphEndpoint, path?: string) => {
     if (
       selectedGraphEndpoint === graphEndpoint &&
       RouteUtils.getEnabledParaTimes().includes(selectedGraphEndpoint as unknown as ParaTime)
     ) {
-      navigate(`/${graphEndpoint}`)
+      navigate(`/${graphEndpoint}${path ? `/${path}` : ''}`)
 
       return
     }
```

Comment:
At some point I want to refactor this into links, as tested out in POC https://github.com/oasisprotocol/explorer-frontend-poc/pull/17/commits/0e39ab772d726406d7e9f3a69d5824c740b588e2

Reply:
nice thanks!